src/main/java/de/doubleslash/keeptime/view/ManageWorkController.java

```diff
@@ -0,0 +1,276 @@
+         }
+      });
+
+      // set text of selected value
+      projectComboBox.setConverter(new StringConverter<Project>() {
+         @Override
+         public String toString(final Project project) {
+            if (project == null) {
+               return null;
+            } else {
+               return project.getName();
+            }
+         }
+
+         @Override
+         public Project fromString(final String string) {
+            // ignores String and gets selected Value of ComboBox
+            return projectComboBox.getValue();
+         }
+      });
+
+      // needs to set again because editable is ignored from fxml because of custom preselection of current Project
+      projectComboBox.setEditable(true);
+
+      projectComboBox.valueProperty().addListener(
+            (final ObservableValue<? extends Project> observable, final Project oldValue, final Project newValue) -> {
+               if (newValue == null) {
+                  return;
+               }
+
+               selectedProject = newValue;
+               comboChange = true;
+               // needed to avoid exception on empty textfield https://bugs.openjdk.java.net/browse/JDK-8081700
+               Platform.runLater(() -> {
+                  projectComboBox.getEditor().selectAll();
+                  setColor(projectComboBox, newValue.getColor());
+               });
+            }
+
+      );
+
+      projectComboBox.getEditor().textProperty().addListener(new ChangeListener<String>() {
+
+         Boolean isValidProject = true;
+
+         @Override
+         public void changed(final ObservableValue<? extends String> observable, final String oldValue,
+               final String newValue) {
+
+            // ignore selectionChanges
+            if (comboChange) {
+               comboChange = false;
+               isValidProject = true;
+
+               return;
+            }
+
+            // is necessary to not autoselect same Project if Project was selected
+            if (Boolean.TRUE.equals(isValidProject)) {
+               isValidProject = false;
+               projectComboBox.getSelectionModel().clearSelection();
+            }
+            // needed to avoid exception on empty textfield https://bugs.openjdk.java.net/browse/JDK-8081700
+            Platform.runLater(() -> {
+
+               projectComboBox.hide();
+               projectComboBox.setItems(model.getAllProjects().filtered(project -> ProjectsListViewController
+                     .doesProjectMatchSearchFilter(projectComboBox.getEditor().getText(), project)));
+               if (projectComboBox.getEditor().focusedProperty().get()) {
+                  projectComboBox.show();
+               }
+
+            });
+
+         }
+      });
+
+      // manages Focusbehaviour
+      projectComboBox.getEditor().focusedProperty().addListener((final ObservableValue<? extends Boolean> observable,
+            final Boolean oldIsFocused, final Boolean newIsFocused) -> {
```

Comment:
fyi: you should also be able to omit the types like (observable, oldIsFocused, newIsFocused) ->